client/a8c-for-agencies/components/items-dashboard/item-preview-pane/index.tsx

```diff
@@ -11,20 +11,29 @@ import { FeaturePreviewInterface, PreviewPaneProps } from './types';
 import './style.scss';
 
 export const createFeaturePreview = (
```

Comment:
I think this function is getting too complicated. Can we create our own (dotcom) version in `DotcomPreviewPane` instead?

This way, we don't have to pass  `selectedSiteFeature, setSelectedSiteFeature, selectedSiteFeaturePreview` because they are already available inside `DotcomPreviewPane`. The feature array will be more compact.

What do you think?

Reply:
Addressed by https://github.com/Automattic/wp-calypso/pull/91589/commits/c7437d0fd2d9341790d4dcdf7f2e30e7d1dfc18a 👍